lib/katello/engine.rb

```diff
@@ -105,7 +112,7 @@ def find_assets(args = {})
       end
     end
 
-    initializer "katello.initialize_cp_listener", after: "foreman_tasks.initialize_dynflow" do
```

Comment:
The `:after => "foreman_tasks.initialize_dynflow"` condition was not needed with recent foreman_tasks changes and `on_init` block.